templates/docs/examples/patterns/navigation/_script-sliding-search.js

```diff
@@ -0,0 +1,67 @@
+function initNavigationSearch(element) {
```

Comment:
I haven't looked into details, but how different it is from existing script for search? https://github.com/canonical/vanilla-framework/blob/main/templates/docs/examples/patterns/navigation/_script-search.js

This looks to be share a lot, would it be possible for one script to handle both existing version and sliding version?